applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -246,7 +246,8 @@ static int headset_pres_delay_find(uint8_t index, uint32_t *pres_dly_us)
 		return 0;
 	}
 
-	if (IS_ENABLED(CONFIG_BT_AUDIO_PRES_DELAY_SRCH_PREF_MIN)) {
+	/* If preferred delay from unicast_server is 0 it is up to unicast_server to select delay */
+	if (IS_ENABLED(CONFIG_BT_AUDIO_PRES_DELAY_SRCH_PREF_MIN) && pref_dly_min != 0) {
```

Comment:
We need a check that the end pres_dly_us is in valid range.

Reply:
Discussed offline and it is not needed. The range of the gateway is not necessary to take into account.